app.yaml

```diff
@@ -31,6 +31,14 @@ handlers:
   static_files: static/robots.txt
   upload: static/robots\.txt
 
+- url: /service-worker\.js
+  static_files: static/dist/service-worker.js
+  upload: static/dist/service-worker\.js
+  expiration: "0m"
+  secure: always
+  http_headers:
+    Vary: Accept-Encoding
```

Comment:
I don't think it's needed. I copied from an example, and meant to go back and clean it up. I'll remove it.